app/resources/app.yaml

```diff
@@ -25,3 +25,39 @@ hooks:
                   - create
                   - -f
                   - /var/lib/gravity/resources/install.yaml
+  update:
+    job: |
+      apiVersion: batch/v1
+      kind: Job
+      metadata:
+        name: bandwagon-update
+      spec:
+        parallelism: 1
+        completions: 1
+        template:
+          metadata:
+            name: bandwagon-update
+          spec:
+            restartPolicy: Never
```

Comment:
I'm curious about `Never` as a restart policy. I understand that if the hook is restarted, it will make things much more complicated to implement, but this can happen through a transient etcd error in which case the update is effectively broken.

Reply:
It's the same as in all our other upgrade hooks. It just means that the container will not be restarted in case of error - the job controller will still schedule another pod though.